topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+
+As elastic net penalty with stochastic decent gradient approach induces sparsity in the number of features used in classification, and the best/top features (genes) are likely to represent transcriptional signature of given disease or aberrant activity of the mutated genes in a pathway. 
+
+Each feature (gene) is given a rank and score (negative or positive) depending on its contribution to classification. The positive scored genes are likely to be up-regulated in activated pathway samples and negatively scored genes are likely to be downstream targets of altered pathways. 
+
+In this tutorial, we made series of steps to generate classification models and used those models for predicting pharmacological response or identifying potential biomarkers that may be helpful for treatment of various cancers. Here, we generate models using ERBB2, KRAS, PIK3CA, and AKT11 oncogenes from the ERK/RAS/PI3K signaling axis pathway.  **Have fun!**
+
+## **Pre-installed tutorial tools, datasets and workflows from the docker image**
+> An efficient way to install and run the tutorial using papaa tools is available on docker based galaxy instance that has pre-installed papaa tool-suite as **papaa** under tools section. Additionally this local galaxy instance comes with datasets and workflow for generating PI3K_OG classifier. Instructions to run the docker image is below.
+
+> ### {% icon hands_on %} Hands-on: Tutorial for galaxy docker container installation and running the workflow:
+> 1. Pulling the docker image from docker hub: Open a terminal and type the following command: 
+>       ```docker pull nvk747/galaxy_papaa:0.1.9```
+> 2. For running the docker container use the following command :
+>       ```docker run -p 8888:80 --detach nvk747/galaxy_papaa:0.1.9```
+> 3. Open the following [*http://localhost:8888*](http://localhost:8888) in your web browser.
+> 4. Use *'admin'* as username and *'password'* as password to login as **admin** into the docker galaxy instance.
+> 5. The Workflow **'papaa@0.1.9_PI3K_OG_model_tutorial'** from workflow section to run PI3K_OG tutorial is pre-installed.
+> 6. Instructions to get datasets from shared library is given in Hands-on: Data upload.
+{: .hands_on}
+
+## **Get data for analysis from classifier to pharmocological correlations**
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [PAPAA-Zenodo](https://zenodo.org/record/4306639#.X9FJF-lKgZE) or from the docker shared data library (`library`->`GTN-Material`->`PAPAA:PanCancer Aberrant Pathway Activity Analysis`->`DOI: 10.5281/zenodo.4306639#.X9FJF-lKgZE`
```

Comment:
you can leave the template variables here, then it will automatically stay correct even if you ever change the tutorial name etc

```suggestion
> 2. Import the files from [PAPAA-Zenodo](https://zenodo.org/record/4306639#.X9FJF-lKgZE) or from the docker shared data library (`GTN - Material -> {{ page.topic_name }} -> {{ page.title }} )
```

Reply:
@shiltemann yes will keep them. Thank you once again for taking your time for taking a look and for suggestions. I will update and get back to you.